modules/ipi-install-bmc-addressing-for-fujitsu.adoc

```diff
@@ -0,0 +1,53 @@
+// This is included in the following assemblies:
+//
+// installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc
+[id='bmc-addressing-for-fujitsu_{context}']
+
+= BMC addressing for Fujitsu
```

Comment:
Lets make it clear that this is for Fujitsu HW not Fujitsu the customer/partner of ours.